kallisto/kallisto.wdl

```diff
@@ -0,0 +1,65 @@
+version 1.0
+# -------------------------------------------------------------------------------------------------
+# Package Name: https://pachterlab.github.io/kallisto
+# Task Summary: Quantifies abundances of transcripts from RNA-Seq data using pseudo-alignment
+# Tool Name: Kallisto
+# Documentation: https://pachterlab.github.io/kallisto/manual
+# -------------------------------------------------------------------------------------------------
+
+task Kallisto {
+  input {
+    File kallisto
+    File kallisto_index
+    String output_path = "."
+    Array[File] fastqs
```

Comment:
Can we change this to `fastq_1` and `fastq_2` like  [this](https://github.com/chopdgd/bfx-tools-wdl/blob/develop/novoalign/NovoAlign.wdl#L44)